common/io/SelectServerTest.cpp

```diff
@@ -121,14 +121,19 @@ class SelectServerTest: public CppUnit::TestFixture {
   }
 
   void Terminate() {
+    OLA_WARN << "Terminate called";
```

Comment:
I'd left it in deliberately; if the order that executes in is important, don't we want to ensure it's working like that if there's another failure?